src/Microsoft.TestPlatform.ObjectModel/Client/Events/TestRunCompleteEventArgs.cs

```diff
@@ -106,4 +106,10 @@ public TestRunCompleteEventArgs(ITestRunStatistics stats, bool isCanceled, bool
     /// </summary>
     [DataMember]
     public IDictionary<string, object> Metrics { get; set; }
+
+    /// <summary>
+    /// Gets or sets the collection of discovered extensions.
+    /// </summary>
+    [DataMember]
+    public IDictionary<string, ISet<string>> DiscoveredExtensions { get; set; }
```

Comment:
Avoid null by doing init to empty dictionary?

Reply:
Done.